### DISCUSSION TITLE
[QUESTION] Export Longhorn volume data to NFS.

### DISCUSSION THREAD

**Discussion by hamadodene** [longhorn/longhorn#7068] [General]
## Question
Is there a way to export data from a Longhorn volume to NFS? We have multiple PODs (around 20) and need to export this data locally to an NFS mount. Is there any utility that makes this process straightforward and efficient?

**Comment by derekbit** [comment]
If the volume is mounted,  you can export Longhorn mountpoint by NFS protocol on the host.
